homeassistant/components/tplink/strings.json

```diff
@@ -21,27 +21,27 @@
       },
       "user_auth_confirm": {
         "title": "Authenticate",
-        "description": "The device requires authentication, please input your credentials below.",
+        "description": "The device requires authentication, please input your TP-Link credentials below.",
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
       "discovery_auth_confirm": {
         "title": "Authenticate",
-        "description": "The device requires authentication, please input your credentials below.",
+        "description": "The device requires authentication, please input your TP-Link credentials below.",
         "data": {
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
       },
       "reauth": {
         "title": "[%key:common::config_flow::title::reauth%]",
-        "description": "The device needs updated credentials, please input your credentials below."
+        "description": "The device needs updated credentials, please input your TP-Link credentials below."
       },
       "reauth_confirm": {
         "title": "[%key:common::config_flow::title::reauth%]",
-        "description": "The device needs updated credentials, please input your credentials below.",
+        "description": "The device needs updated credentials, please input your TP-Link credentials below.",
```

Comment:
We could make these references since they are all the same to save translators some work

ex:
https://github.com/home-assistant/core/blob/b60ca4260c7afb0140cc6dd50e0be6e10e64888a/homeassistant/components/homekit/strings.json#L12

Reply:
Yes, we definitely want to do that :+1: